tests/test_api.py

```diff
@@ -2,9 +2,13 @@
 from unittest.mock import MagicMock, call, patch
 
 import pytest
+from dateutil.relativedelta import relativedelta
 
 from ami_deprecation_tool import api, configmodels
 
+one_month_ago = datetime.now() - relativedelta(months=1)
+six_months_ago = datetime.now() - relativedelta(months=6)
```

Comment:
i would capitalize these vars: e.g. `one_month_ago` -> `ONE_MONTH_AGO`. python doesn't really do constants, but still good to express intent